assets/js/components/chess-board.js

```diff
@@ -41,29 +68,71 @@ class ChessBoard extends React.Component {
   renderRanks() {
     const board = this.getBoard();
 
-    return _.map(Object.keys(board).reverse(), (rankId) => {
+    return _.map(this.ranks(), rankId => {
```

Comment:
I'm assuming some kind of preprocessor is used since your using ES6 features, otherwise your just depending on a modern browser. Either way, could we use the native Array map function here instead of lodash?

Reply:
Yeah, we tried that first. It doesn't work in PhantomJS 😞